HorribleDownloader/parser.py

```diff
@@ -84,7 +84,7 @@ def _get_html(showid: int, limit: int, show_type: str) -> str:
             response = requests.get(api, params=query)
             # the limit is counted in number of episodes (or batches)
             # because each page contains 12 episodes, we must divide it by 12.
-            if response.text == stop_text or query["nextid"] >= limit // 12:
+            if response.text == stop_text or query["nextid"] > limit // 12:
```

Comment:
I would change it to `(limit - 1) // 12` because then it wont send another request. I might be wrong as this was not tested.

Reply:
pretty smart, I'll change it